test-suite/non-functional-tests/test-scenario-1/dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM openjdk:17-jdk-alpine
+
+ARG JMETER_VERSION="5.5"
+ARG JMETER_HOME="/opt/apache-jmeter-${JMETER_VERSION}"
+ARG JMETER_BIN="${JMETER_HOME}/bin"
+ARG MIRROR_HOST=http://mirrors.ocf.berkeley.edu/apache/jmeter
+ARG JMETER_DOWNLOAD_URL="${MIRROR_HOST}/binaries/apache-jmeter-${JMETER_VERSION}.tgz"
+ARG JMETER_PLUGINS_DOWNLOAD_URL="http://repo1.maven.org/maven2/kg/apc"
+ARG JMETER_LIB_FOLDER="${JMETER_HOME}/lib/"
+ARG JMETER_PLUGINS_FOLDER="${JMETER_HOME}/lib/ext/"
+ARG INBOUND_CERT_PATH="/jmeter-config/certs"
+
+ARG FACADE_URL 
+ARG FACADE_PORT 
+ARG INBOUND_URL
+ARG INBOUND_PORT
+
+RUN apk update  \
+    && apk upgrade  \
+    && apk add --update openjdk8-jre tzdata curl unzip bash  \
+    && rm -rf /var/cache/apk/*  \
+    && mkdir -p /tmp/dependencies/ \
```

Comment:
Minor feedback, `mkdir` can accept multiple directories.

Also `-p` will create intermediate directories, so `mkdir -p ./jmeter-config/certs` will create `./jmeter-config` if it doesn't already exist.

`mkdir -p /tmp/dependeices/ /opt ./jmeter-logs ./jmeter-config/certs`

Reply:
ahh was not aware of that, have simplified in all the files